packages/spor-theme-react/src/foundations/sizes.ts

```diff
@@ -0,0 +1,34 @@
+import { spacing } from "./spacing";
```

Comment:
These were missing, so I copied in the original code from Chakra UI. We might want to override these at some point.